lib/Re2.fx

```diff
@@ -0,0 +1,1234 @@
+/*
+    This file is a part of ficus language project.
+    See ficus/LICENSE for the licensing terms
+*/
+
+/*
+    Re2 library wrapper.
+
+    Usage: 
+    1.) Re2 library must be installed in system, see installation instructions there:
+    https://github.com/google/re2/wiki/Install
+
+    Sometimes(E.g.,, on Debian 10) it's also needed to run
+        sudo ldconfig 
+    after installation for creating appropriate soft links for shared libraries.
+
+    2.) Before compilation set environment variable FICUS_LINK_LIBRARIES to "-lre2" for providing compiler info about dependencies.
+        e.g, in bash:
+        export FICUS_LINK_LIBRARIES="-lre2"
+    3.) Use -c++ flag on compilation.
+
+
+    See https://github.com/google/re2/wiki/Syntax re2.h of Re2 source file for more detailed information about syntax and options. 
+    Wrapper function signatures have ficus-dependent specialization. They are described there.
+*/
+
+exception BadRegexp : string
+ccode 
+{
+    #include <re2/re2.h>
+    #include <vector>
+
+    int _fx_M3Re2FM14make_BadRegexpE1S(fx_str_t* arg0, fx_exn_t* fx_result); //[TODO] : Exception, declared before ccode block must be declared in program, translated to C program before this ccode for avoiding such insets. 
```

Comment:
put `FX_EXTERN_C` before `int _fx_M3...`

Reply:
Done.
But. Using compiler -c++ flag directly will cause compilation errors on this string.
Deleting correspondent pragma change nothing, it's still error.
Should we disable -c++ flag?